src/mobile/src/main/java/tupperdate/android/ui/home/profile/ProfileGeneral.kt

```diff
@@ -0,0 +1,181 @@
+                modifier = Modifier.shade().fillMaxSize(),
+                fadeIn = true,
+                contentScale = ContentScale.Crop,
+            )
+            Row(
+                modifier = Modifier.align(Alignment.BottomStart)
+                    .padding(16.dp)
+            ) {
+                Text(
+                    text = recipe.title,
+                    style = MaterialTheme.typography.caption,
+                    color = Color.White
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun ProfileRecap(
+    name: String,
+    email: String,
+    image: Any,
+    onEditClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Row(
+        modifier = modifier.fillMaxWidth().padding(end = 3.dp, top = 16.dp, bottom = 16.dp)
+            .height(56.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        ProfilePicture(
+            image = image,
+            highlighted = false,
+            modifier = Modifier.size(56.dp)
+        )
+
+        Column(modifier = Modifier.padding(horizontal = 16.dp)) {
+            Text(text = name, style = MaterialTheme.typography.subtitle1)
+            Text(text = email, style = MaterialTheme.typography.subtitle2)
+        }
+        Spacer(modifier = Modifier.weight(1f, true))
+        IconButton(
+            onClick = onEditClick,
+            modifier = Modifier.height(19.dp)
+        ) {
+            Icon(
+                imageVector = vectorResource(id = R.drawable.ic_editrecipe_edit),
+            )
+        }
+    }
+}
+
+@Preview
+@Composable
+fun ProfileGeneralPreview() {
+    val reList = listOf(
+        RecipeApi.Recipe(
+            "Red lobster",
+            "https://www.theflavorbender.com/wp-content/uploads/2019/01/How-to-cook-Lobster-6128-700x1049.jpg",
+            "In the Santa Monica way"
+        ),
+        RecipeApi.Recipe(
+            "Red lobster",
+            "https://www.theflavorbender.com/wp-content/uploads/2019/01/How-to-cook-Lobster-6128-700x1049.jpg",
+            "In the Santa Monica way"
+        ),
+        RecipeApi.Recipe(
+            "Red lobster",
+            "https://www.theflavorbender.com/wp-content/uploads/2019/01/How-to-cook-Lobster-6128-700x1049.jpg",
+            "In the Santa Monica way"
+        ),
+        RecipeApi.Recipe(
+            "Red lobster",
+            "https://www.theflavorbender.com/wp-content/uploads/2019/01/How-to-cook-Lobster-6128-700x1049.jpg",
+            "In the Santa Monica way"
+        ),
+    )
+    TupperdateTheme {
+        ProfileGeneral(name = "Aloy", email = "chieftain@banuk",
+            image = "https://pbs.twimg.com/profile_images/1257192502916001794/f1RW6Ogf_400x400.jpg",
+            location = "Song's Edge",
+            userRecipes = reList,
+            onEditClick = {},
+            onLocationChange = {},
+        onNewRecipeClick = {})
+    }
+}
+
+private const val recipeCardWidth = 120
+private const val recipeCardHeight = 160
```

Comment:
Please use some `Dp` rather than some `Int` here, and use PascalCase rather than camelCase.